DigitalLearningSolutions.Web/Helpers/DateHelper.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers
+{
+    public static class DateHelper
+    {
+        public static string StandardDateFormat = "dd/MM/yyyy";
+
+        public static string StandardDateAndTimeFormat = "dd/MM/yyyy hh:mm";
+    }

```

Comment:
There are some date formats added by Jonathan as part of the reports work that will need to be moved across too (see latest master)